docs/plugins/search/meilisearch.md

```diff
@@ -0,0 +1,108 @@
+---
+icon: https://docsearch.algolia.com/img/favicon.ico
+---
+
+# meilisearch
+
+<NpmBadge package="@vuepress/plugin-meilisearch" />
+
+Integrate [MeiliSearch](https://www.meilisearch.com/) into VuePress, which can provide search to your documentation site.
+
+## Usage
+
+```bash
+npm i -D @vuepress/plugin-meilisearch@next
+```
+
+```ts
+import { MeiliSearchPlugin } from 'vuepress-plugin-meilisearch2'
+
+export default {
+  plugins: [
+    meilisearchPlugin({
+      // Configuration options
+      host: '',
+      apiKey: '',
+      indexUid: '',
+    }),
+  ],
+}
+```
+
+## Options
+
+### host
+
+- Type: `string`
+
+- Required: `true`
+
+- Details:
+
+  Provide the HTTP address of the MeiliSearch API
+
+### apiKey
+
+- Type: `string`
+
+- Required: `true`
+
+- Details:
+
+  API key generated by MeiliSearch
+
+### indexUid
+
+- Type: `string`
+
+- Required: `true`
+
+- Details:
+
+  Specify the index name used for searching
+
+### translations
+
+- Type: `DocSearchTranslations`
+
+- Required: `false`
+
+- Details:
+
+  button and modal
+
+### hotKeys
+
+- Type: `false | string[]`
+
+- Required: `false`
+
+- Details:
+
+  Default keys are `['ctrl+k', 's', '/']`, Set to `false` to disable default keys.
+
+### debounceDuration
+
+- Type: `number | false`
+
+- Required: `false`
+
+- Details:
+
+- Duration to wait between keystores to determine whether a search should happen or not. Defaults to `200`. Set to `false` to disable debouncing.
```

Comment:
Can we set `0` to disable debouncing? Is this an official usage from meilisearch? Introducing a union type while unnecessary could be confusing.